pallets/runtime/common/src/runtime.rs

```diff
@@ -335,7 +336,7 @@ macro_rules! misc_pallet_impls {
             type DepositPerByte = polymesh_runtime_common::DepositPerByte;
             type CallStack = [pallet_contracts::Frame<Self>; 5];
             type WeightPrice = pallet_transaction_payment::Pallet<Self>;
-            type WeightInfo = pallet_contracts::weights::SubstrateWeight<Self>;
+            type WeightInfo = polymesh_weights::pallet_contracts::SubstrateWeight;
```

Comment:
We are not able to run the benchmarks for all functions. We get an UnauthorizedCaller` error, which I believe is to due to the CallFilter in this runtime.

Reply:
We can either use a different `CallFilter` for the benchmark build, or just allow those calls (most likely just System.remark or other simple Substrate pallet calls).

Reply:
The benchmarks are trying to deploy a contract from an account that isn't linked to an identity.  So it isn't a primary key or secondary key (i.e. has no call permissions) and no CDD claim.